runbooks/runbook.md

```diff
@@ -0,0 +1,118 @@
+# UPP - Content Collection Unfolder
+
+Content Collection Unfolder finds added/deleted collection members and lead articles through the relations-api.
+
+## Code
+
+content-collection-unfolder
+
+## Primary URL
+
+https://upp-prod-delivery-glb.upp.ft.com/__content-collection-unfolder/
+
+## Service Tier
+
+Platinum
+
+## Lifecycle Stage
+
+Production
+
+## Delivered By
+
+content
+
+## Supported By
+
+content
+
+## Known About By
+
+- dimitar.terziev
+- elitsa.pavlova
+- kalin.arsov
+- hristo.georgiev
+- georgi.ivanov
+- elina.kaneva
+- robert.marinov
+- tsvetan.dimitrov
+
+## Host Platform
+
+AWS
+
+## Architecture
+
+This service forwards mapped content collections to `content-collection-rw-neo4j`. If the response is 200, it retrieves the elements in the collection from `document-store-api` and places them in Kafka on the `PostPublicationEvents` topic so that notifications will be created for each one of them.
+
+## Contains Personal Data
+
+No
+
+## Contains Sensitive Data
+
+No
+
+## Dependencies
+
+* upp-neo4j-cluster
+* upp-kafka
```

Comment:
Different dependancies are listed in the README:

> Dependencies are:
> 
> relations-api
> content-collection-rw-neo4j
> document-store-api
> kafka

Reply:
Fixed.